server/bangs.json

```diff
@@ -26,8 +37,32 @@
   "bm": {
   	"any":"https://www.bing.com/mapspreview/?q={{{s}}}&cc={{{lang}}}"
   },
+  
+
+  "imdb": {
+    "any": "http://www.imdb.com/find?q={{{s}}}&s=all"
+  },
+
 
+  "py": {
+    "any": "https://docs.python.org/3/search.html?q={{{s}}}"
+  },
+  "python": {
+    "any": "https://docs.python.org/3/search.html?q={{{s}}}"
+  },
+  "pip": {
```

Comment:
"pip" is just one client tool (but having the bang is fine), let's add "pypi" too